### ISSUE TITLE
UrlMapsTo has two url builds, only one is used

### ISSUE THREAD

**Issue by neuhoffm** [MattHoneycutt/SpecsFor#57]
The function has this code:
```
var url = BuildUrlFromExpression(helper.ViewContext.RequestContext, helper.RouteCollection, action);
var expectedUrl = MvcWebApp.BaseUrl + helper.BuildUrlFromExpression(action);
return (Browser.Url == expectedUrl);
```
https://github.com/MattHoneycutt/SpecsFor/blob/master/SpecsFor.Mvc/MvcWebApp.cs#L152 

The url variable has the more robust code that supports areas where as expectedUrl just uses the standard out of the box html helper. Should expectedUrl be removed and use url for the comparison instead? I can do the PR just wanted to make sure that I wasn't missing anything.

**Comment by MattHoneycutt** [comment]
Hmm, that's a great question, actually.  It looks like it would need to be
a combination of the two lines (since we still need the MvcWebApp.BaseUrl
portion, I think).  I'm good with a pull request that makes this change if
you are up for it.

**Comment by MattHoneycutt** [comment]
Fixed with 4.5.0.
